.github/workflows/netlify.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: 8.0
+          php-version: '8.1'
```

Comment:
This is not necessary and unrelated to "Update composer dependencies and allow minor version updates to Twig".